### ISSUE TITLE
🐛thread 'main' panicked at 'regex string should be pre-tested: Error(-208, invalid backref number/name)'

### ISSUE THREAD

**Issue by kissge** [dandavison/delta#202]
I found delta today and it looks nice.
This is what I've experienced right now.
If any additional info is necessary, please guide me.

Repro steps:
1. git clone https://github.com/kissge/SAO
2. Edit .gitconfig as follows:
```
[log]
    abbrevCommit = true
    decorate = short
[core]
    pager = delta --dark --file-color="#4444ff" --hunk-color="#4444ff"
[interactive]
    diffFilter = delta --color-only
```
3. `git log -p` on master HEAD (162596a)

Output:
```
thread 'main' panicked at 'regex string should be pre-tested: Error(-208, invalid backref number/name)', /home/*****/.cargo/registry/src/github.com-1ecc6299db9ec823/syntect-4.1.0/src/parsing/regex.rs:70:17
stack backtrace:
   0:     0x7f1a6bd34025 - <unknown>
   1:     0x7f1a6bd55a5c - <unknown>
   2:     0x7f1a6bd31843 - <unknown>
   3:     0x7f1a6bd36640 - <unknown>
   4:     0x7f1a6bd3639c - <unknown>
   5:     0x7f1a6bd36c41 - <unknown>
   6:     0x7f1a6bd3682b - <unknown>
   7:     0x7f1a6bd54411 - <unknown>
   8:     0x7f1a6bd54233 - <unknown>
   9:     0x7f1a6bcba1d7 - <unknown>
  10:     0x7f1a6bcc1e28 - <unknown>
  11:     0x7f1a6bcc5648 - <unknown>
  12:     0x7f1a6bbf1d0b - <unknown>
  13:     0x7f1a6bbe480d - <unknown>
  14:     0x7f1a6bbe1c3a - <unknown>
  15:     0x7f1a6bbec1b5 - <unknown>
  16:     0x7f1a6bbed2b0 - <unknown>
  17:     0x7f1a6bd36703 - <unknown>
  18:     0x7f1a6bd3ca97 - <unknown>
  19:     0x7f1a6bd3715c - <unknown>
  20:     0x7f1a6bbed112 - <unknown>
  21:     0x7f1a6b9ae002 - __libc_start_main
  22:     0x7f1a6bbd010e - <unknown>
  23:                0x0 - <unknown>
```

Info:
delta 0.1.1
git version 2.26.2
Linux ***** 4.4.0-18362-Microsoft #836-Microsoft Mon May 05 16:04:00 PST 2020 x86_64 GNU/Linux
(Arch Linux on WSL)

**Comment by dandavison** [comment]
Thanks for opening this @kissge. It looks like this was the same as https://github.com/sharkdp/bat/issues/650. This should be fixed in master by #200  (updated the syntect library version). Would you be able to confirm?

**Comment by kissge** [comment]
Thanks @dandavison ! Just as you say, it is now fixed as of delta 0.2.0.
